Source/Scene/Camera.js

```diff
@@ -338,6 +343,67 @@ function updateCameraDeltas(camera) {
   }
 }
 
+/**
+ * Initialize the camera pivot point in the DOM
+ */
+Camera.prototype._initPivotElement = function () {
+  var camera = this;
+
+  // Initialize the camera pivot element, if needed
+  if (document.getElementById("cesium_camera_pivot") === null) {
+    camera._domPivot = document.createElement("div");
+    camera._domPivot.setAttribute(
```

Comment:
Ideally, the style of this DOM element should be something the user can configure.